tests/testsuite/build.rs

```diff
@@ -1863,11 +1916,47 @@ fn cargo_rustc_current_dir_foreign_workspace_dep() {
     // Verify it works from a different workspace
     foo.cargo("test -p baz")
         .masquerade_as_nightly_cargo(&["CARGO_RUSTC_CURRENT_DIR"])
-        .with_stdout_contains("running 1 test\ntest baz_env ... ok")
+        .with_stdout_data(str![[r#"
```

Comment:
We already assert libtest output elsewhere, so I think this is fine to check the entire output here.